### PR TITLE
New JS bounties - Mik317

### PR DESCRIPTION
**Pull request by Jmimoni**

### CODE REVIEW

**Comment by JamieSlome** [comment]
@Jmimoni - there are a few mistakes in here, we can go through them tomorrow morning to resolve them before we merge it in. Looks good otherwise.

File: bounties/npm/cordova-serve/1/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Overview
+
+[cordova-serve](https://github.com/zaach/jison) provides a JavaScript API to serve up a Cordova application in the browser.
```
Comment (JamieSlome) [inline bounties/npm/cordova-serve/1/README.md:3]: The URL for this is for jison but the package is cordova-serve?

File: bounties/npm/cordova-serve/1/bounty.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "ForkURL": "https://github.com/bitovi/launchpad/",
```
Comment (JamieSlome) [inline bounties/npm/cordova-serve/1/bounty.json:2]: This is in the cordora-serve directory but the fork url is for bitovi launchpad?

File: bounties/npm/express-brute/1/bounty.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
-    "ForkURL": "https://github.com/418sec/express-brute",
+    "ForkURL": "https://github.com/clns/node-commit-msg/",
```
Comment (JamieSlome) [inline bounties/npm/express-brute/1/bounty.json:2]: This doesn't seem right? We are changing the fork URL for express-brute?

File: bounties/npm/jison/1/bounty.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
-    "ForkURL": "https://github.com/418sec/jison",
+    "ForkURL": "https://github.com/bitovi/launchpad/",
```
Comment (JamieSlome) [inline bounties/npm/jison/1/bounty.json:2]: This bounty was resolved and should be closed/deleted.

File: bounties/npm/strider-git/1/README.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Overview
+[strider-grit](https://www.npmjs.com/package/strider-git) allows strider to use any git repository for a project.
```
Comment (JamieSlome) [inline bounties/npm/strider-git/1/README.md:2]: Is it spelled strider-grit or strider-git?

File: bounties/npm/strider-git/1/bounty.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+    "ForkURL": "https://github.com/Strider-CD/strider-git",
```
Comment (JamieSlome) [inline bounties/npm/strider-git/1/bounty.json:2]: git or grit?

File: bounties/npm/strider-git/1/vulnerability.json [hunk 7]
```diff
@@ -0,0 +1,50 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-16",
+    "AffectedVersionRange": "*",
+    "Summary": "Remote Code Execution",
+    "Author": {
+        "Username": "",
```
Comment (JamieSlome) [inline bounties/npm/strider-git/1/vulnerability.json:7]: Add Mik317

File: bounties/npm/windows-edge/1/README.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
+# Overview
+[safer-eval](https://www.npmjs.com/package/safer-eval) is a safer approach for eval in node and browser.
```
Comment (JamieSlome) [inline bounties/npm/windows-edge/1/README.md:2]: safer-eval, this is inside windows-edge?

File: bounties/npm/windows-edge/1/bounty.json [hunk 9]
```diff
@@ -0,0 +1,7 @@
+{
+    "ForkURL": "https://github.com/418sec/safer-eval",
```
Comment (JamieSlome) [inline bounties/npm/windows-edge/1/bounty.json:2]: this should be windows-edge?

**Comment by Jmimoni** [comment]
Sweet man!

Jake Mimoni

File: index.json [hunk 10]
```diff
@@ -1 +1 @@
...
...AffectedVersionRange":"*","DisclosureDate":"2020-02-13","Bounty":{"Credit":590,"Cash":25}},{"Registry":"packagist","PackageName":"teampass","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"PHP","VulnerabilityDescription":"Unauthorized File Access","Severity":"7.5","AffectedVersionRange":"*","DisclosureDate":"2020-04-29","Bounty":{"Credit":750,"Cash":25}},{"Registry":"maven","PackageName":"argonaut","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"Java","VulnerabilityDescription":"Denial of Service (DoS)","Severity":"4.9","AffectedVersionRange":"*","DisclosureDate":"2020-04-10","Bounty":{"Credit":490,"Cash":25}},{"Registry":"maven","PackageName":"openid-connect-java-spring-server","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"Java","VulnerabilityDescription":"Cross Site Scripting (XSS)","Severity":"6.1","AffectedVersionRange":"*","DisclosureDate":"2020-02-03","Bounty":{"Credit":610,"Cash":25}},{"Registry":"maven","PackageName":"gwtupload","PackageVulnerabilityID":"1","CodebasePrimaryLanguage":"Java","VulnerabilityDescription":"Denial of Service (DoS)","Severity":"7.5","AffectedVersionRange":"*","DisclosureDate":"2020-06-01","Bounty":{"Credit":750,"Cash":25}}]
```
Comment (JamieSlome) [inline index.json:1]: I would be careful merging this into master as a lot of the data is incorrect and potentially may break existing bounties.

File: bounties/npm/launchpad/1/README.md [hunk 11]
```diff
@@ -0,0 +1,6 @@
+# Overview
+
+[jison](https://github.com/zaach/jison) is a package that provides an API for creating parsers in JavaScript.
```
Comment (JamieSlome) [inline bounties/npm/launchpad/1/README.md:3]: jison or launchpad?

File: bounties/npm/cordova-serve/1/vulnerability.json [hunk 12]
```diff
@@ -0,0 +1,55 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-16",
+    "AffectedVersionRange": "*",
+    "Summary": "Remote Code Execution",
+    "Author": {
```
Comment (adam-nygate) [inline bounties/npm/cordova-serve/1/vulnerability.json:6]: Add Mik as the author

File: bounties/npm/cordova-serve/1/vulnerability.json [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+    "References": [
+        {
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/apache/cordova-serve/blob/master/src/browser.js#L111"
+        },
+        {
+            "Description": "Permalink",
+            "URL": "https://github.com/apache/cordova-serve/blob/master/src/browser.js#L118"
+        }
+    ]
```
Comment (adam-nygate) [inline bounties/npm/cordova-serve/1/vulnerability.json:54]: Permalinks need to be in a separate array

File: bounties/npm/dzzoffice/1/vulnerability.json [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+        "URL": "https://github.com/zyx0814/dzzoffice/",
+        "Codebase": [
+            "JavaScript"
+        ]
+    },
+    "References": [
+        {
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/zyx0814/dzzoffice/blob/master/admin/setting/index.php#L200"
+        }
```
Comment (adam-nygate) [inline bounties/npm/dzzoffice/1/vulnerability.json:49]: Separate array

File: bounties/npm/launchpad/1/vulnerability.json [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/bitovi/launchpad/blob/master/lib/local/instance.js#L12"
+        },
+        {
+            "Description": "Permalink",
+            "URL": "https://github.com/bitovi/launchpad/blob/master/lib/local/instance.js#L93"
+        },
+        {
+            "Description": "Permalink",
+            "URL": "https://github.com/bitovi/launchpad/blob/master/lib/local/instance.js#L97"
```
Comment (adam-nygate) [inline bounties/npm/launchpad/1/vulnerability.json:56]: Separate array

File: bounties/npm/mod-auth-pubtkt/1/bounty.json [hunk 16]
```diff
@@ -0,0 +1,7 @@
+{
+    "ForkURL": "https://github.com/manuelkasper/mod_auth_pubtkt/",
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/bounty.json:2]: Empty ForkURL

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json [hunk 17]
```diff
@@ -0,0 +1,51 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-16",
+    "AffectedVersionRange": "*",
+    "Summary": "Open Redirect",
+    "Author": {
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/vulnerability.json:6]: Add author

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+        "Username": "",
+        "Name": ""
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "mod-auth-pubtkt",
+        "URL": ""
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/vulnerability.json:13]: Add URL

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+    },
+    "CWEs": [
+        {
+            "ID": "601",
+            "Description": ""
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/vulnerability.json:18]: Add description

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json [hunk 20]
```diff
@@ -0,0 +1,51 @@
...
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "7.5"
+    },
+    "CVEs": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/vulnerability.json:37]: Remove empty CVE

File: bounties/npm/mod-auth-pubtkt/1/vulnerability.json [hunk 21]
```diff
@@ -0,0 +1,51 @@
...
+        "URL": "https://github.com/manuelkasper/mod_auth_pubtkt/",
+        "Codebase": [
+            "JavaScript"
+        ]
+    },
+    "References": [
+        {
+            "Description": "GitHub Permalink",
+            "URL": "https://github.com/manuelkasper/mod_auth_pubtkt/blob/master/php-login/login.php#L105"
+        }
```
Comment (adam-nygate) [inline bounties/npm/mod-auth-pubtkt/1/vulnerability.json:49]: Separate array

File: bounties/npm/strider-git/1/vulnerability.json [hunk 22]
```diff
@@ -0,0 +1,50 @@
...
+        "Name": "Michele Romano"
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "strider-grit",
+        "URL": ""
```
Comment (adam-nygate) [inline bounties/npm/strider-git/1/vulnerability.json:13]: Missing

File: bounties/npm/strider-git/1/vulnerability.json [hunk 23]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    "CWEs": [
+        {
+            "ID": "77",
+            "Description": ""
```
Comment (adam-nygate) [inline bounties/npm/strider-git/1/vulnerability.json:18]: Missing

File: bounties/npm/strider-git/1/vulnerability.json [hunk 24]
```diff
@@ -0,0 +1,50 @@
...
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "7.8"
+    },
+    "CVEs": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/strider-git/1/vulnerability.json:37]: Remove

File: bounties/npm/strider-git/1/vulnerability.json [hunk 25]
```diff
@@ -0,0 +1,50 @@
...
+    "Repository": {
+        "URL": "https://github.com/Strider-CD/strider-git",
+        "Codebase": [
+            "JavaScript"
+        ]
+    },
+    "References":   {
+        "Description": "GitHub Permalink",
+        "URL": "https://github.com/Strider-CD/strider-git/blob/master/lib/index.js#L151"
+    }
```
Comment (adam-nygate) [inline bounties/npm/strider-git/1/vulnerability.json:48]: Needs to be an array of objects

File: bounties/npm/windows-edge/1/bounty.json (hunk 9 above)
Comment (adam-nygate) [reply bounties/npm/windows-edge/1/bounty.json:2]: Empty

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 26]
```diff
@@ -0,0 +1,50 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-16",
+    "AffectedVersionRange": "*",
+    "Summary": "Arbitrary Code Execution",
+    "Author": {
+        "Username": "",
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:7]: Missing

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 27]
```diff
@@ -0,0 +1,50 @@
...
+        "Name": "Michele Romano"
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "windows-edge",
+        "URL": ""
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:13]: Missing

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 28]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    "CWEs": [
+        {
+            "ID": "77",
+            "Description": ""
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:18]: Missing

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 29]
```diff
@@ -0,0 +1,50 @@
...
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "7.3"
+    },
+    "CVEs": [
+        ""
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:37]: Remove

File: bounties/npm/windows-edge/1/vulnerability.json [hunk 30]
```diff
@@ -0,0 +1,50 @@
...
+    "Repository": {
+        "URL": "https://github.com/eugeneware/windows-edge",
+        "Codebase": [
+            "JavaScript"
+        ]
+    },
+    "References":   {
+        "Description": "GitHub Permalink",
+        "URL": "https://github.com/eugeneware/windows-edge/blob/master/index.js#L8"
+    }
```
Comment (adam-nygate) [inline bounties/npm/windows-edge/1/vulnerability.json:48]: Separate array

**Comment by adam-nygate** [comment]
Also can you check for GitHub Issues for these vulnerabilities?

**Pull request closed without merging**